cli/src/cli/sign.rs

```diff
@@ -0,0 +1,34 @@
+use crate::cli::keygen::get_root_seed;
+use holochain_core_types::error::HcResult;
+use lib3h_sodium::secbuf::SecBuf;
+
+
+pub fn sign(root_seed_string: Option<String>, message: String) -> HcResult<String> {
+    let mut root_seed = get_root_seed(root_seed_string, &String::from(""), false)?;
+    let mut revocation_keypair = root_seed.generate_revocation_key()?;
```

Comment:
@willemolding, We may need to be able to pass in the revocation key. Because the revocation key can be changed, so the revocation key won't have a fixed derivation path.

Also, I don't think it's wise to always have to use your master seed for revocation.